assets/js/components/Eula/Eula.test.jsx

```diff
@@ -9,12 +9,24 @@ import { setEulaVisible } from '@state/settings';
 import Eula from '.';
 
 describe('Eula component', () => {
-  it('should render correctly', () => {
+  it('should render community eula correctly', () => {
     const [statefulEula, store] = withState(<Eula />);
 
     store.dispatch(setEulaVisible());
 
     render(statefulEula);
     expect(screen.getByRole('button')).toHaveTextContent('Accept');
+    expect(screen.getByText(/Trento Community/)).toBeTruthy();
   });
+
+  // it('should render premium eula correctly', () => {
+  //   const [statefulEula, store] = withState(<Eula />);
+
+  //   store.dispatch(setEulaVisible());
+  //   store.dispatch(setIsPremium());
+
+  //   render(statefulEula);
+  //   expect(screen.getByRole('button')).toHaveTextContent('Accept');
+  //   expect(screen.getByText(/Trento Premium/)).toBeTruthy();
+  // });
```

Comment:
Why is this commented? I think it's fine :smile:

Reply:
LOL you outran me

Reply:
It was failing, because I didn't include the `intersection-observer`, now it works